roles/k8s_best_practices_certsuite/tasks/tests.yml

```diff
@@ -0,0 +1,80 @@
+---
+- name: Create a directory to place config files
+  ansible.builtin.file:
+    path: "{{ kbpc_certsuite_dir }}/config_files"
+    state: directory
+    mode: '0755'
+
+- name: Create a directory to place result files
+  ansible.builtin.file:
+    path: "{{ kbpc_certsuite_dir }}/result_files"
+    state: directory
+    mode: '0755'
+
+- name: Template tnf_config.yml
+  ansible.builtin.template:
+    src: templates/tnf_config.yml.j2
+    dest: "{{ kbpc_certsuite_dir }}/config_files/tnf_config.yml"
+    mode: "750"
```

Comment:
should it be 0755 as well?

Reply:
Yes, it could be, this one I didn't modify it because it was defined in that way in cnf_cert role, now it's updated